docs/metrics/extending-the-sdk/README.md

```diff
@@ -29,6 +29,12 @@ not covered by the built-in exporters:
   done via `OpenTelemetry.SuppressInstrumentationScope`.
 * Exporters receives a batch of `Metric`, and each `Metric`
   can contain 1 or more `MetricPoint`s.
+  The exporter should perform all actions (e.g. serializing etc.) with
+  the `Metric`s and `MetricsPoint`s in the batch before returning control from
+  `Export`, once the control is returned, the exporter can no longer make any
+  assumptions about the state of the batch or anything inside it.
+* Exporters should use `Activity.TagObjects` collection instead of
```

Comment:
this line is to be removed.

Reply:
i don't remember adding this. merge conflict maybe?
regardless, removing it now